File: 2019/Day 1/Task 1-2.cpp
#include <iostream>
#include <fstream>

using namespace std;

int main ()
{
	ifstream fin;
	int sum = 0, x;
	fin.open("Task 1-1.txt");
	if(fin.is_open() )
	{
		while( fin >> x )
		{
			
			x = x / 3 - 2;
			while( x > 0 ){
				sum += x;
				x = x / 3 - 2;
			}
		}
		cout << sum << endl;
		fin.close();
	}
	else
		cout << "Can't open the file";
	return 0;
}
	

File: 2019/Day 1/Task 1-2.py
with open("Task 1-1.txt") as f:
	masses_list = []
	for line in f:
		masses_list.append(int(line.strip()))
fuel = 0
for mass in masses_list:
	mass = mass//3-2
	while mass > 0:
		fuel += mass
		mass = mass//3-2
	
print(fuel)
